src/everdev/motd.ts

```diff
@@ -0,0 +1,36 @@
+import * as path from "path";
+
+import { everdevHome } from "../core/index";
+import { run, StringTerminal, compareVersions, touch} from '../core/utils'
+
+/*
+ * Checks if this is the first run in the last 24 hours
+ * and touches the `.lastrun` file
+ */
+function isFirstRun(): boolean {
+    const lastrunFile = path.resolve(everdevHome(), ".lastrun");
+    const lastrunTS: Date | undefined = touch(lastrunFile);
+    return !lastrunTS || lastrunTS.getTime() < Date.now() - 24 * 3600 * 1000;
+}
+
+export async function motd(pkgName: string, pkgVer: string): Promise<string> {
```

Comment:
Could you give readable name to this function (and file)?

Reply:
I want to have some kind of function that can inform the user about various events, not just new releases of `everdev`, so I chose the name "motd" because this acronym is well known as "message of the day".